src/libraries/System.Threading.Channels/ref/System.Threading.Channels.csproj

```diff
@@ -10,6 +10,9 @@
   <ItemGroup Condition="!$(TargetFramework.StartsWith('netstandard'))">
     <Compile Include="System.Threading.Channels.netcoreapp.cs" />
   </ItemGroup>
+  <ItemGroup Condition="!$(TargetFramework.StartsWith('netstandard')) OR '$(TargetFramework)' == 'netstandard2.1'">
```

Comment:
This condition is a little hard to comprehend.  Can you make it to be just !netstandard1.3 and !netstandard2.0?

Reply:
done